src/xml_handler.py

```diff
@@ -0,0 +1,165 @@
+import xml.etree.ElementTree as ET
```

Comment:
this should be a class, like  `CageGeometry`  with read, write and update functions